src/components/ExpansionPanel/Body/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Body = ({ children, ...passedProps }) => (
```

Comment:
`children` is one of the props you can spread to other components without having to explicitly pass them. If we are going to keep this pass-thru component, you can clean it up to just one line:

```js
const Body = ({ ....props }) => <div {...props} />;
```